liquidprompt

```diff
@@ -3310,62 +3417,68 @@ _lp_default_theme_prompt_data() {
         LP_SHLVL=
     fi
 
-    if _lp_python_env_color; then
-        LP_VENV=" $lp_python_env_color"
+    if _lp_python_env_color ; then
+        LP_VENV="$lp_python_env_color"
     else
         LP_VENV=
     fi
 
     if _lp_node_env_color; then
-        LP_NODE_VENV=" $lp_node_env_color"
+        LP_NODE_VENV="$lp_node_env_color"
     else
         LP_NODE_VENV=
     fi
 
     if _lp_ruby_env_color; then
-        LP_RUBY_VENV=" $lp_ruby_env_color"
+        LP_RUBY_VENV="$lp_ruby_env_color"
     else
         LP_RUBY_VENV=
     fi
 
     if _lp_kubernetes_context_color; then
-        LP_KUBECONTEXT=" $lp_kubernetes_context_color"
+        LP_KUBECONTEXT="$lp_kubernetes_context_color"
     else
         LP_KUBECONTEXT=
     fi
 
     if _lp_terraform_env_color; then
-        LP_TFSPACE=" $lp_terraform_env_color"
+        LP_TFSPACE="$lp_terraform_env_color"
     else
         LP_TFSPACE=
     fi
 
     if _lp_container_color; then
-        LP_CONTAINER=" $lp_container_color"
+        LP_CONTAINER="$lp_container_color"
     else
         LP_CONTAINER=
     fi
 
     if _lp_software_collections_color; then
-        LP_SCLS=" $lp_software_collections_color"
+        LP_SCLS="$lp_software_collections_color"
     else
         LP_SCLS=
     fi
 
     if _lp_aws_profile_color; then
-        LP_AWS_PROFILE=" $lp_aws_profile_color"
+        LP_AWS_PROFILE="$lp_aws_profile_color"
     else
         LP_AWS_PROFILE=
     fi
 
+    if _lp_cmake_color; then
+        LP_CMAKE="$lp_cmake_color"
```

Comment:
Add a space to this one for consistency
```suggestion
        LP_CMAKE=" $lp_cmake_color"
```

Reply:
It's in the dev section, so it should be handled by `LP_MARK_DEV_MID`.